docs/content/users/install/ddev-installation.md

```diff
@@ -26,49 +24,52 @@ It's easiest to think of DDEV installation or upgrade as the installation of a s
     mkcert -install
     ```
 
-    ### install_ddev.sh install script
+    ## Install Script
+
+    !!!tip
+        The install script works on macOS, Linux, and Windows WSL2.
 
-    On macOS, Linux and Windows WSL2 you can use the install_ddev.sh script
 
-    Use this line on your terminal to download, verify, and install (or upgrade) ddev using the [install_ddev.sh script](https://github.com/drud/ddev/blob/master/scripts/install_ddev.sh). Note that this works with both amd64 and arm64 architectures, including Surface Pro X with WSL2 and 64-bit Raspberry Pi OS. It also works with macOS Apple Silicon M1 machines.
+    Run the [install script](https://github.com/drud/ddev/blob/master/scripts/install_ddev.sh) to install or update DDEV. It downloads, verifies, and sets up the `ddev` binary:
 
     ```
     curl -fsSL https://raw.githubusercontent.com/drud/ddev/master/scripts/install_ddev.sh | bash
     ```
 
-    The installation script can also take a version argument in order to install a specific version or a prerelease version. For example,
+    You can include an `-s` argument to install a specific release or a prerelease version:
```

Comment:
Taking this suggestion as-is, but I always waffle on these. Your suggestion is technically correct, but when I read the sentence aloud “an” seems like the more natural word to use there because it sounds like “include an ess version argument”.